docs/supported_versions.md

```diff
@@ -0,0 +1,205 @@
+# OCP Version Support
+
+## MCE hub (management or hosting) Cluster
+[Multi-Cluster Engine](https://docs.openshift.com/container-platform/4.17/architecture/mce-overview-ocp.html) (MCE), which is available through OpenShift's OperatorHub, requires specific OCP versions for the Management Cluster to remain in a supported state.
+
+Each version documents its own support matrix. For example, 
+
+- [MCE 2.7](https://access.redhat.com/articles/7086906)
+- [MCE 2.6](https://access.redhat.com/articles/7073030)
+- [MCE 2.5](https://access.redhat.com/articles/7056007)
+- [MCE 2.4](https://access.redhat.com/articles/7027079)
+
+As a heuristic, OCP versions MCE supports are:
+
+- The latest, yet to be released version (N+1) of OpenShift
```

Comment:
Does it mean that MCE 2.6 should deploy hosted 4.17? Or it is beginning from MCE 2.7?